spec/README.md

```diff
@@ -0,0 +1,40 @@
+# GovStack Sandbox
+
+The GovStack Sandbox is a unified environment for all the GovStack and digital government
+ecosystem actors to discover, understand, engage, experiment, innovate, and build digital
+government solutions using the [GovStack platform](http://govstack.gitbook.io/).
+
+The sandbox is not just a playground, but rather a publicly available kit of tools, including:
+* cloud infrastructure configurations
+* container images
+* CI/CD pipeline configurations
+* Helm charts for building blocks.
+ 
+This means that you have the flexibility to take these repositories and set up your own sandbox environment.
+
+The fully operational environment offers a comprehensive API that public and private agencies can use to mimic the
+characteristics exhibited by the GovStack platform and environment on a real-time basis to help simulate responses from 
+all the systems an application interfaces with. This especially enables government agencies and digital government 
+players to experiment with innovative GovTech products or services within a well-defined space and duration. The design 
+of the GovStack sandbox encapsulates the business logic and data necessary to represent multiple GovStack capabilities 
+such as APIs, building blocks, use cases and transaction flows. This helps developers and digital government agencies 
+to achieve a more holistic view of the GovStack.
+## Get started
+
+The sandbox is reusable and replicable, which means you can set up your own GovStack sandbox within your own environment.
+
+To get more details check out the [infrastructure documentation](https://github.com/GovStackWorkingGroup/sandbox-infra). 
```

Comment:
Presumably, we only want one place to document the sandbox? Is it going to be this repo or the other?

Reply:
This particular repository is the main documentation for Sandbox.
The [infrastructure repository](https://github.com/GovStackWorkingGroup/sandbox-infra) is the part of the sandbox that is responsible for setting up the environment for the sandbox instance.

I updated this part too.

Reply:
The sandbox team is currently focused on testing GovStack's theory of running BB candidates in a single environment. As a result, the team provides vendor-independent configuration.
Anyone else can take this configuration and check it out for themselves.
Perhaps in the future a public playground will be built.
This is one of the reasons why we have many repositories.